genesis-builder/src/main.rs

```diff
@@ -126,6 +126,22 @@ fn fix_aptos_packages() -> anyhow::Result<()> {
         }
         write(move_toml_file, content)?;
     }
+
+    // Object module should also friend the new u256 version of the fungible store
+    let object_move_file = &APTOS_FRAMEWORK_DIR
+        .join("aptos-framework")
+        .join("sources")
+        .join("object.move");
+    let mut content = read_to_string(object_move_file)?;
+    content = content.replace(
+        "friend aptos_framework::primary_fungible_store;",
+        "friend aptos_framework::primary_fungible_store;\n    friend aptos_framework::primary_fungible_store_u256;"
```

Comment:
Any better way to write this line?

Reply:
Alternatively the following, which is at least not as long:
```rust
content = content.replace(
        "friend aptos_framework::primary_fungible_store;",
        "friend aptos_framework::primary_fungible_store;
    friend aptos_framework::primary_fungible_store_u256;"
```

Reply:
I think this is ok as is. The version with the line break would also be fine if you prefer it.